CONTRIBUTING.md

```diff
@@ -97,6 +111,13 @@ When submitting a PR, please keep the following guidelines in mind:
 
 For an example of a properly formatted PR, take a look at https://github.com/wildfly-security/wildfly-elytron/pull/1532
 
+### Documentation
+Contributors are encouraged to keep documentation up-to-date along with code changes. If your changes impact user-facing features, update the relevant documentation files in directory.
+Community documentation can be found in the [wildfly repository](https://github.com/wildfly/wildfly/tree/main/docs)
+
+## FAQ for New Contributors
+Check out our [Frequently Asked Questions](https://fjuma.github.io/wildfly-elytron/blog/frequently-asked-questions-new-contributors/) for New Contributors.
```

Comment:
This link should be https://wildfly-security.github.io/wildfly-elytron/blog/frequently-asked-questions-new-contributors/

Reply:
I'm going to merge this one and create a good-first-issue for updating the link.